modules/collector-requirements.adoc

```diff
@@ -12,6 +12,21 @@ It connects to Sensor to report this information.
 To install Collector on systems that have Unified Extensible Firmware Interface (UEFI) and that have Secure Boot enabled, you must use eBPF probes because kernel modules are unsigned, and the UEFI firmware cannot load unsigned packages. Collector identifies Secure Boot status at the start and switches to eBPF probes if required.
 ====
 
+[discrete]
+== Collection requirements
+
+To use the `CORE_BPF` collection method, the base kernel must support BTF, and the BTF file must be available to collector.
+In general, the kernel version must be newer than 4.18 and the `CONFIG_DEBUG_INFO_BTF` configuration option must be set.
+
+To manually check a node for compatibility you can run the following command on the node:
+
+[source,terminal]
+----
+$ ls /sys/kernel/btf/vmlinux
+----
+
+If the `vmlinux` file exists, the kernel has BTF support and `CORE_BPF` is configurable.
```

Comment:
What about the ringbuf map dependency, is it too much details for this document?

Reply:
hmm that is a good point - this complicates the matter a little in that the ringbuf map arrived in 5.8. I'll make a note of it and indicate that this is backported in RHEL kernels

Reply:
In fact, since ringbuf is not controlled by a config option - I'll go with something like 

```
the kernel version must be newer than 5.8 (4.18 for RHEL nodes) and the `CONFIG_DEBUG_INFO_BTF` configuration option must be set. 
```

WDYT?